core/runtime/op_driver/op_results.rs

```diff
@@ -263,21 +263,64 @@ impl<C: OpMappingContext> OpResult<C> {
   }
 }
 
-#[derive(Debug, Serialize)]
-#[serde(rename_all = "camelCase")]
-pub struct OpError {
-  #[serde(rename = "$err_class_name")]
-  class_name: &'static str,
-  message: String,
-  code: Option<&'static str>,
+#[derive(Debug)]
+pub struct OpError(Box<dyn JsErrorClass>);
+
+impl std::error::Error for OpError {
+  fn source(&self) -> Option<&(dyn Error + 'static)> {
+    todo!()
```

Comment:
could this `todo!` cause trouble? regardless, could probably just leave it the default impl for now instead of panicking

Reply:
removed the source impl as not necessary